elastic/logs/challenges/synthetic-source-recovery.json

```diff
@@ -0,0 +1,99 @@
+{% import "rally.helpers" as rally %}
+{
+  "name": "synthetic-source-recovery",
+  "description": "Indexes logs for a specified time period and volume per day and then runs the recovery api fetching translog operations for a given target index.",
+  "default": false,
+  "schedule": [
+    {% include "tasks/index-setup.json" %},
+    {# serverless-shard-recovery-marker-start #}{%- if build_flavor != "serverless" -%}
+    ,
+    {
+      "name": "add-retention-lease",
+      "operation": {
+        "operation-type": "raw-request",
+        "path": {{ "/%s/seq_no/add_retention_lease" | format(p_recovery_target) | tojson }},
+        "method": "PUT"
+      },
+      "warmup-iterations": 0,
+      "iterations": 1
+    },
+    {%- endif -%}{# serverless-shard-recovery-marker-end #}
+    {
+      "name": "bulk-index",
+      "operation": {
+        "operation-type": "raw-bulk",
+        "param-source": "processed-source",
+        "time-format": "milliseconds",
+        "profile": "fixed_interval",
+        "init-load": true,
+        "bulk-size": {{ p_bulk_size }},
+        "detailed-results": true
+      },
+      "clients": {{ p_bulk_indexing_clients }},
+      "ignore-response-error-level": "{{error_level | default('non-fatal')}}"
+    },
+    {
+      "name": "refresh-after-index",
+      "index": "logs-*",
+      "operation": "refresh"
+    },
+    {
```

Comment:
I don't think we need to these operations related to merging? It just makes this challenge run longer?